guides/source/action_view_overview.md

```diff
@@ -118,93 +157,40 @@ would produce something like:
 </div>
 ```
 
-Below is a full-length RSS example actually used on Basecamp:
-
-```ruby
-xml.rss("version" => "2.0", "xmlns:dc" => "http://purl.org/dc/elements/1.1/") do
-  xml.channel do
-    xml.title(@feed_title)
-    xml.link(@url)
-    xml.description "Basecamp: Recent items"
-    xml.language "en-us"
-    xml.ttl "40"
-
-    for item in @recent_items
-      xml.item do
-        xml.title(item_title(item))
-        xml.description(item_description(item)) if item_description(item)
-        xml.pubDate(item_pubDate(item))
-        xml.guid(@person.firm.account.url + @recent_items.url(item))
-        xml.link(@person.firm.account.url + @recent_items.url(item))
-        xml.tag!("dc:creator", item.author_name) if item_has_creator?(item)
-      end
-    end
-  end
-end
-```
-
-### Jbuilder
-
-[Jbuilder](https://github.com/rails/jbuilder) is a gem that's
-maintained by the Rails team and included in the default Rails `Gemfile`.
-It's similar to Builder but is used to generate JSON, instead of XML.
-
-If you don't have it, you can add the following to your `Gemfile`:
-
-```ruby
-gem "jbuilder"
-```
-
-A Jbuilder object named `json` is automatically made available to templates with
-a `.jbuilder` extension.
-
-Here is a basic example:
-
-```ruby
-json.name("Alex")
-json.email("alex@example.com")
-```
-
-would produce:
-
-```json
-{
-  "name": "Alex",
-  "email": "alex@example.com"
-}
-```
-
-See the [Jbuilder documentation](https://github.com/rails/jbuilder#jbuilder) for
-more examples and information.
+See [Builder documentation](https://github.com/jimweirich/builder) for more examples.
 
-### Template Caching
+### Caching
```

Comment:
The type of caching mentioned here (compiling to a method) is unrelated to the more commonly known Action View caching (storing a Template or a part of it in Redis for example). To avoid confusion I think we should rename this (and not mention Fragment Caching here).
```suggestion
### Template Compilation
```